toolchains/flatc/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+"""
+Toolchain for FlatBuffers compiler. Provides the path to the flatc executable.
+"""
+
+# Workaround for https://github.com/bazelbuild/bazel/issues/14009, remove when migrating to Bazel 9.
+load(":toolchain.bzl", "resolve_toolchain")
+
+toolchain_type(
+    name = "toolchain_type",
+    visibility = ["//visibility:public"],
+)
+
+resolve_toolchain(
+    name = "resolved_toolchain",
+    visibility = ["//visibility:public"],
+)
```

Comment:
I don't fully understand what for we need this workaround. The `resolved_toolchain` rule seems to not be used anywhere, since we don't use `genrule` or other rules that need concrete `toolchain` instead of `toolchain_type`. Is this required implicitly or in the subsequent PR?

Also, once we migrate to bazel 9, what exactly we would want to do here? Just remove the file?

Reply:
Yep, true, it is no longer needed.